dist/docker/bin/docker

```diff
@@ -6,16 +6,16 @@ if [ ${#@} -ne 0 ] && [ "${@#"-h"}" = "" ]; then
 USAGE="Magento Cloud Docker
 
 \033[33mArguments:\033[0m
-  pull          pull latest images
-  up            destroy, re-create and start containers
-  down          destroy containers
-  build         build application
-  deploy        deploy application
-  redeploy      re-build and re-deploy application
-  bash          connect to bash
-  stop          stop containers
-  start         start containers
-  restart       restart containers
+  pull              pull latest images
+  up                destroy, re-create and start containers
+  down              destroy containers
+  bash              connect to bash
+  stop              stop containers
+  start             start containers
+  restart           restart containers
+  ece-build         build application
+  ece-deploy        deploy application
+  ece-redeploy      re-build and re-deploy application
```

Comment:
We do not follow BC in command naming and we are ok with renaming them without leaving aliases for old ones?

Reply:
It was not released yet. Only in `develop` branch

Reply:
I see